lib/opentypesvg/fonts2svg.py

```diff
@@ -193,6 +188,31 @@ def processFonts(font_paths_list, hex_colors_list, outputFolderPath, options):
     final_message(filesSaved)
 
 
+def viewbox_settings(font_paths_list, options):
+    tt_font = ttLib.TTFont(font_paths_list[0])
+    if not options.adjust_view_box_to_glyph:
+        # Gather the fonts' UPM. For simplicity, it's assumed that all fonts have
+        # the same UPM value. If fetching the UPM value fails, default to 1000.
+        try:
+            upm = tt_font['head'].unitsPerEm
+        except KeyError:
+            upm = 1000
+        return """viewBox="0 -{} {} {}">\n""".format(upm, upm, upm)
+    else:
+        try:
+            hhea = tt_font['hhea']
+            height = hhea.ascender + abs(hhea.descender) + hhea.lineGap # hhea.descender is negative
+            width = hhea.advanceWidthMax
+            y = -hhea.ascender
```

Comment:
Use the x/yMin/Max values from the `head` table instead.

Reply:
Also, only one try/except is needed; move the if/else branching to the inside of it.